src/Azure.Functions.Cli/Diagnostics/ConsoleTelemetryClientFactory.cs

```diff
@@ -1,27 +0,0 @@
-﻿using Azure.Functions.Cli.Common;
-using Microsoft.ApplicationInsights;
-using Microsoft.ApplicationInsights.Extensibility.Implementation;
-using Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel.Implementation;
-using Microsoft.Azure.WebJobs.Logging.ApplicationInsights;
-using Microsoft.Extensions.Logging;
-using System;
-
-namespace Azure.Functions.Cli.Diagnostics
-{
-    internal class ConsoleTelemetryClientFactory : DefaultTelemetryClientFactory
-    {
-        public ConsoleTelemetryClientFactory(string instrumentationKey, SamplingPercentageEstimatorSettings samplingSettings, Func<string, LogLevel, bool> filter)
-            : base(instrumentationKey, samplingSettings, filter)
-        {
-        }
-
-        public override TelemetryClient Create()
-        {
-            TelemetryClient client = base.Create();
-
-            client.Context.GetInternalContext().SdkVersion = $"azurefunctionscoretools: {Constants.CliVersion}";
```

Comment:
You're missing this logic now. You can do this by registering a TelemetryClient (factory) with DI. See how we do this in Script here: https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/ScriptHostBuilderExtensions.cs#L164-L173

Reply:
Thanks! Planned to have that added after your PR and didn't.  Will have it updated.